easybuild/easyconfigs/c/cuSPARSELt/cuSPARSELt-0.6.0.6-CUDA-12.1.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'Tarball'
+
+name = 'cuSPARSELt'
+version = '0.6.0.6'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://docs.nvidia.com/cuda/cusparselt/index.html'
+description = """NVIDIA cuSPARSELt is a high-performance CUDA library dedicated to general matrix-matrix operations in
+which at least one operand is a sparse matrix"""
+
+toolchain = SYSTEM
+
+local_arch = {'arm64': 'sbsa'}.get(ARCH, ARCH)
+source_urls = ['https://developer.download.nvidia.com/compute/cusparselt/redist/libcusparse_lt/linux-%s/' % local_arch]
+sources = ['libcusparse_lt-linux-%s-%%(version)s-archive.tar.xz' % local_arch]
+checksums = [{
+    'libcusparse_lt-linux-x86_64-%(version)s-archive.tar.xz':
+        'da20d7a6d7a6674cb6e991439eaacd4661631c71a2f1026c41fba115c5c9c8b5',
+    'libcusparse_lt-linux-sbsa-%(version)s-archive.tar.xz':
+        '46b258c2c333f7324e03bdf6b13576be6153b1218fb531025631650e36515f45',
```

Comment:
@Micket I think this is the checksum of `libcusparse_lt-linux-x86_64-0.3.0.3-archive.tar.xz`

checksum of `ibcusparse_lt-linux-sbsa-0.6.0.6-archive.tar.xz` is `51db8182444b4e2c125397ea95e4996c022ef791a87ce7a7389204e77617297f`

Reply:
Fixed in #20216